scipy/stats/tests/test_stats.py

```diff
@@ -3967,12 +3969,47 @@ def test_ttest_ind_exact_alternative(self):
         assert_equal(res_l_ab.pvalue[~mask] + res_g_ba.pvalue[~mask],
                      res_2_ab.pvalue[~mask])
 
+    def test_ttest_ind_exact_selection(self):
+        # test the various ways of activating the exact test
+        np.random.seed(0)
+        N = 3
+        a = np.random.rand(N)
+        b = np.random.rand(N)
```

Comment:
similar comment here, can/should we use Generator if newer numpy available?

Reply:
Are there benefits? It doesn't matter what the numbers are, so I don't think it's worth branching depending on what version of NumPy is present. When `np.Generator` is available in the lowest supported version, we can start using that, until then, I'll only use `Generator` in tests when I want to make sure that the code works with `Generator`. (But I here I just need some numbers.)

Reply:
ok.